mmv1/third_party/terraform/utils/compute_instance_helpers.go.erb

```diff
@@ -262,6 +302,45 @@ func expandNetworkInterfaces(d TerraformResourceData, config *Config) ([]*comput
 	return ifaces, nil
 }
 
+// This func is temp. It should be deleted once gVNIC is GA for resource_compute_instance_template 
+func expandNetworkInterfaces2(d TerraformResourceData, config *Config) ([]*computeBeta.NetworkInterface, error) {
+	configs := d.Get("network_interface").([]interface{})
+	ifaces := make([]*computeBeta.NetworkInterface, len(configs))
+	for i, raw := range configs {
+		data := raw.(map[string]interface{})
+
+		network := data["network"].(string)
+		subnetwork := data["subnetwork"].(string)
+		if network == "" && subnetwork == "" {
+			return nil, fmt.Errorf("exactly one of network or subnetwork must be provided")
+		}
+
+		nf, err := ParseNetworkFieldValue(network, d, config)
+		if err != nil {
+			return nil, fmt.Errorf("cannot determine self_link for network %q: %s", network, err)
+		}
+
+		subnetProjectField := fmt.Sprintf("network_interface.%d.subnetwork_project", i)
+		sf, err := ParseSubnetworkFieldValueWithProjectField(subnetwork, subnetProjectField, d, config)
+		if err != nil {
+			return nil, fmt.Errorf("cannot determine self_link for subnetwork %q: %s", subnetwork, err)
+		}
+
+		ifaces[i] = &computeBeta.NetworkInterface{
+			NetworkIP:     data["network_ip"].(string),
+			Network:       nf.RelativeLink(),
+			Subnetwork:    sf.RelativeLink(),
+			AccessConfigs: expandAccessConfigs(data["access_config"].([]interface{})),
+			AliasIpRanges: expandAliasIpRanges(data["alias_ip_range"].([]interface{})),
+<% unless version == 'ga' -%>
+			NicType:       expandNicType(data["nic_type"].(interface{})),
```

Comment:
data["nic_type"] is already an interface, we don't have to convert here. And in the original `expandNetworkInterfaces`